File: lista-de-exercicios/exercicio-3/Jogador.ts
import { Escolhas } from './Jokenpo.js';
import { JogadorBase } from './Jokenpo.js';


export class Jogador implements JogadorBase {
    private escolha: Escolhas;

    public constructor (escolha: Escolhas) {
        this.escolha = escolha;
    }

    public getEscolha(): Escolhas {
        return this.escolha;
    }

    public setEscolha(escolha: Escolhas) {
        this.escolha = escolha;
    }
}

File: lista-de-exercicios-java/src/main/java/org/example/exercicio03/Jogador.java
package org.example.exercicio03;

public class Jogador implements JogadorBase {
    private Escolhas escolha;

    public Jogador(Escolhas escolha) {
        this.escolha = escolha;
    }

    @Override
    public Escolhas getEscolha() {
        return escolha;
    }
}
